ServerCore/Pages/Teams/CreateTeam.razor

```diff
@@ -14,21 +14,42 @@
                 </div>
                 @if (Event.AllowsRemoteTeams)
                 {
+                    // If either remote & local has reached the cap then show a message to that effect and default the team to the one that isn't full
                     <div class="form-group">
                         <label>
-                        Team will be playing remotely
-                        <table>
-                            <tr>
-                                <td style="width:50px;vertical-align:top;">
-                                    <InputCheckbox @bind-Value="TeamModel.IsRemoteTeam" />
-                                </td>
+                            This event allows both local and remote teams.
+                            @if(Event.LockChangesToRemoteStatus)
+                            {
+                                <div> Once the team is created this can only be changed by contacting the organizers.</div>
+                            }
+                            <table>
+                                <tr>
+                                    <InputSelect @bind-Value="TeamModel.IsRemoteTeam">
+                                        <option value=null>Select An Option</option>
+                                        @if (LocalTeamsFull)

```

Comment:
Does this work?

`<option @(LocalTeamsFull ? "disabled" : "") value=false>Local</option>`